packages/primitives/src/Menu/Menu.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { Box } from '../Box'
+
+type MenuProps = {
+  children: React.ReactNode
+  'aria-label'?: string
+  dismissLabel?: string
+  onDismiss?: () => void
+} & React.ComponentProps<typeof Box>
+
+export const Menu = React.forwardRef<HTMLUListElement, MenuProps>(function Menu(
+  { children, variantKey = 'menu', variant, ...props },
```

Comment:
```suggestion
  { children, variantKey = 'menus', variant, ...props },
```

I'm not necessarily saying that this change needs to be made but I think we need to define whether variantKeys should be singular or plural.

Reply:
Good point, yes that makes sense, plural for the variantKey 👍